setup.py

```diff
@@ -153,10 +116,16 @@ def run(self):
         sys.argv.append(name.replace('.', '_').replace('-', '_'))
 
 setup(
-    name="pytcg", version='0.0.0.1', description="A Python interface to libtcg and TCG IR",
-    url='https://github.com/angr-tcg/pytcg',
+    name="pytcg",
+    version='0.0.0.1',
+    description="A Python interface to libtcg and TCG IR",
+    url='https://github.com/angr/pytcg',
+    maintainer = 'pwnslinger',
```

Comment:
Ok... Are you volunteering to be the point of contact for issues going forward?

Reply:
yes, if you want also please let me know to add your name too.